.github/workflows/comment.yml

```diff
@@ -0,0 +1,59 @@
+name: comment
+on:
+  workflow_run:
+    workflows: ['build']
+    types: [completed]
+jobs:
+  pr_comment:
+    if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
+    runs-on: ubuntu-latest
+    timeout-minutes: 35
```

Comment:
uh?

Reply:
`github.event.workflow_run.event == 'pull_request'` make sure that build is triggered by PR rather than push or some others (maybe manually run in furture).
`github.event.workflow_run.conclusion == 'success'` make sure build is succeeded, artifacts been produced.

Reply:
`timeout-minutes` has been removed